pkg/cluster/px/px.go

```diff
@@ -659,6 +687,153 @@ func (ops *pxClusterOps) getDaemonSetReadyTimeout() (time.Duration, error) {
 	return daemonsetReadyTimeout, nil
 }
 
+func (ops *pxClusterOps) wipePXKvdb() error {
+	logrus.Info("attempting to parse kvdb info from Portworx daemonset")
+	endpoints, opts, err := ops.parseKvdbFromDaemonset()
+	if err != nil {
+		return err
+	}
+
+	logrus.Infof("creating kvdb client for: %v", endpoints)
+	kvdbInst, err := getKVDBClient(endpoints, opts)
+	if err != nil {
+		return err
+	}
+
+	logrus.Infof("deleting Portworx kvdb tree: %s for endpoint(s): %v", pxKvdbPrefix, endpoints)
+	return kvdbInst.DeleteTree("")
```

Comment:
I think we should delete only the cluster that this daemonset is for.
```
kv.DeleteTree("clusterID")
```
You will have to probably parse "-c" argument as well.

Reply:
fixed